conda-store-server/pyproject.toml

```diff
@@ -69,7 +69,7 @@ dependencies = [
   # pinning them; conda-lock depends on this, but also doesn't pin the setuptools
   # version. See https://github.com/pypa/setuptools/issues/4478 for details
   "setuptools<70",
-  "sqlalchemy<2.0a0",
+  "sqlalchemy>=2.0,<2.1",
```

Comment:
what's the incompatibility with 2.1?

Reply:
I think there shouldn't be any issue based on the [changelog](https://docs.sqlalchemy.org/en/21/changelog/changelog_21.html). But, I never tested this (sqlalchemy 2.1 is still in development) . Was setting this restriction out of an abundance of caution.